automatic/sourcetree/update.ps1

```diff
@@ -0,0 +1,33 @@
+import-module au
+. "$PSScriptRoot\update_helper.ps1"
+
+$releases = 'https://www.sourcetreeapp.com/update/windowsupdates.txt'
+
+function global:au_SearchReplace {
+    @{
+        'tools\chocolateyInstall.ps1' = @{
+            "(^[$]packageName\s*=\s*)('.*')"= "`$1'$($Latest.PackageName)'"
```

Comment:
just replace `[$]` here with `\s*` and it should work when you move that section.